pkgs/development/libraries/nss/default.nix

```diff
@@ -18,16 +27,16 @@ let
   #       It will rebuild itself using the version of this package (NSS) and if
   #       an update is required do the required changes to the expression.
   #       Example: nix-shell ./maintainers/scripts/update.nix --argstr package cacert
-  version = "3.64";
-  underscoreVersion = builtins.replaceStrings ["."] ["_"] version;
+  version = "3.70";
```

Comment:
This should probably update nss_latest instead

Reply:
Made the relevant change in #137236.